rpcs3/Emu/Cell/Modules/cellPad.cpp

```diff
@@ -63,12 +64,51 @@ void pad_info::save(utils::serial& ar)
 	ar(max_connect, port_setting);
 }
 
+void cell_PadEnqueueConnectionChanges(std::array<s32, CELL_MAX_PADS> statuses, usz locked = true)
```

Comment:
I don't see a reason why this should be CELL_MAX_PADS.
It's never using more than CELL_PAD_MAX_PORT_NUM